zipkin-tracer.gemspec

```diff
@@ -42,8 +42,13 @@ Gem::Specification.new do |s|
   s.add_development_dependency 'faraday', '~> 0.8'
   s.add_development_dependency 'timecop', '~> 0.8'
   s.add_development_dependency 'webmock', '~> 1.22'
+  s.add_development_dependency 'mutant-rspec'
+  s.add_development_dependency 'byebug'
 
   # manually install the gem depending on the adapter you want to use
   s.add_development_dependency 'hermann', '~> 0.25'
   s.add_development_dependency 'scribe', '~> 0.2.4'
+  s.add_development_dependency 'benchmark-ips'
+  s.add_development_dependency 'rbtrace'
```

Comment:
those gems should be in a group in the Gemfile (see previous comments) that you'd only need if you want to run the benchmarks.
at the very least these shouldn't be in that "block" because then the comment at line 48 doesn't make sense anymore.